### ISSUE TITLE
Make the system settings filter more useful

### ISSUE THREAD

**Issue by SnowCreative** [modxcms/revolution#16193]
### Summary
Right now, the filter field for system settings seems to only search the key names and descriptions.

### Why is it needed?
I often don't know the name of the Key, and would like to search the Name field as well. For example, searching for "browser" pulls up the Media Browser settings, but searching for "media browser" pulls up nothing.

### Suggested solution(s)
Add the "Name" field what the filter searches.

**Comment by smg6511** [comment]
I've been working on a larger-scale update to all of the grids with filters that includes the feature you're looking for here. I'm of the mind that, in any given grid, the search field should query all text fields not covered by a dropdown filter.

**Comment by smg6511** [comment]
Closing, as the current release (3.0.5) supports full search of the settings grid.

**Comment by Ruslan-Aleev** [comment]
@smg6511 Search by name doesn't work for me. Are you sure the issue is solved?

![name_search](https://github.com/user-attachments/assets/c6745414-1218-491a-98df-f6db4db7f9ea)

**Comment by smg6511** [comment]
@Ruslan-Aleev Yup, thought I'd covered that! I'll reopen and take a closer look soon. However, the only easy and immediate fix would apply to the non-translated (English) value. This is one of the challenges with how the translations are implemented - they don't exist in the database and cannot be queried. Anyway, I'll put up the quick fix and let's see if someone has an idea on how to deal with searching when in a different language. The only way that comes to mind is to change the way the data is loaded into the grid: It would have to load all records and do all filtering and pagination with the data in-place.

**Comment by Ruslan-Aleev** [comment]
Yes, probably searching by lexicons is a complex and not a priority task. It can be fixed in the distant future =)
